django/db/models/fields/__init__.py

```diff
@@ -2810,7 +2825,6 @@ def _check_primary_key(self):
     def deconstruct(self):
         name, path, args, kwargs = super().deconstruct()
         del kwargs["blank"]
-        kwargs["primary_key"] = True
```

Comment:
That might cause some issues for already generated migrations as for cases where `primary_key` was not specified they'll now turn from `True` to `False` and be detected as changes.